README.md

```diff
@@ -109,21 +109,58 @@ $ gx test ./p2p/<path of module you want to run tests for>
 
 ### Packages
 
-> **WIP**
+> This table is generated using the module [`package-table`](https://github.com/ipfs-shipyard/package-table) with `package-table --data=package-list.json`.
 
 List of packages currently in existence for libp2p:
 
-| Package            | Version | CI                  |
-|--------------------|---------|---------------------|
-| **Transports**                                     |
-| **Connection Upgrades**                            |
-| **Stream Muxers**                                  |
-| **Discovery**                                      |
-| **Crypto Channels**                                |
-| **Peer Routing**                                   |
-| **Content Routing**                                |
-| **Miscellaneous**                                  |
-| **Data Types**                                     |
+| Name | CI | Coverage |
+| ---------|---------|--------- |
+| **Libp2p** |
+| [`interface-libp2p`](//github.com/libp2p/interface-libp2p) | [![jenkins](https://ci.ipfs.team/buildStatus/icon?job=libp2p/interface-libp2p/master)](https://ci.ipfs.team/job/libp2p/job/interface-libp2p/job/master/) | [![codecov](https://codecov.io/gh/libp2p/interface-libp2p/branch/master/graph/badge.svg)](https://codecov.io/gh/libp2p/interface-libp2p) |
```

Comment:
can you add a rule that disables the badges for repos that start with `interface-` so that it doesn't show wrong badges?